unstructured/metrics/table/table_eval.py

```diff
@@ -200,37 +200,55 @@ def process_file(self) -> TableEvaluation:
         predicted_table_data = extract_and_convert_tables_from_prediction(
             file_elements=self.prediction, source_type=self.source_type
         )
-
-        matched_indices = TableAlignment.get_table_level_alignment(
-            predicted_table_data,
-            ground_truth_table_data,
-        )
-        if matched_indices:
+        is_table_in_gt = bool(ground_truth_table_data)
+        is_table_predicted = bool(predicted_table_data)
+        if not is_table_in_gt:
+            # There is no table data in ground truth, you either got perfect score or 0
+            score = 0 if is_table_predicted else np.nan
```

Comment:
I'm thinking if we cover that in aggregation function there:
https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/metrics/evaluate.py#L270

Probably worth checking before merging.